src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -64,8 +65,8 @@ public FindCommand(List<Predicate<Person>> predicates) {
      *                  If false, this command will search on the full list of persons.
      */
     public FindCommand(List<Predicate<Person>> predicates, boolean isChained) {
-        // For now we only support 1 or 2 predicates
-        assert predicates.size() == 1 || predicates.size() == 2;
+        // For now we only support 1, 2 or 3 predicates
+        assert predicates.size() <= 3 : "More than 3 predicates found for find command";
```

Comment:
hmm do you also want to check for `>= 1`?

Reply:
I think so!

Reply:
Hmmm that's true